doc/modules/classes.rst

```diff
@@ -1197,7 +1197,7 @@ See the :ref:`metrics` section of the user guide for further details.
    preprocessing.MaxAbsScaler
    preprocessing.MinMaxScaler
    preprocessing.Normalizer
-   preprocessing.OneHotEncoder
```

Comment:
Looked at the code before I looked at your description. I think your description is a good summary.